### DISCUSSION TITLE
Welcome to Sand-Table Discussions!

### DISCUSSION THREAD

**Discussion by rdudhagra** [rdudhagra/Sand-Table#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ishan0221** [comment]
Hi, great project and looking forward to create my own version! One quick question. If I were to recreate this for a different dimension table would the only changes be...

-  change the size of print bed in the marlin/config.h file

Thanks :)

**Reply by rdudhagra** [reply]
You’ll need to change it there and also in the `.env` file in the website. Make sure your X and Y are correctly set if your table isn’t square (you can use Pronterface or a similar application to manually move your table around without the website).

**Reply by ishan0221** [reply]
Wonderful!
